include/crow/query_string.h

```diff
@@ -362,16 +362,41 @@ namespace crow
 
         }
 
+        /// Get a value from a name, used for `?name=value`.
+        ///
+        /// Note: this method returns the value of the first occurance of the key only, to return all occurances, see \ref get_list().
```

Comment:
`occurance` -> `occurrence`

Reply:
Have I been typing it wrong all this time !?? 😂